assignment1/day 2/calculateFrequency.js

```diff
@@ -0,0 +1,27 @@
+// Problem 2: Complete the calculateFrequency function that takes lowercase string as input and returns frequency of all english alphabet. (using only array, no in-built function)
+function calculateFrequency(string) 
+{
+  // Write your code here
+  var output=[];
```

Comment:
there should be a space before and after the operator in JS code. eg: `a + b` instead of `a+b`